code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -6,21 +6,17 @@
 /mob/living/carbon/xenomorph/flamer_fire_act(burnlevel)
 	if(xeno_caste.caste_flags & CASTE_FIRE_IMMUNE)
 		return
-	if(get_fire_resist() <= 0)
-		return
 	return ..()
 
 /mob/living/carbon/xenomorph/modify_by_armor(damage_amount, armor_type, penetration, def_zone)
-	var/hard_armor_modifier = get_hard_armor(armor_type, def_zone)
-	hard_armor_modifier = hard_armor_modifier - (hard_armor_modifier * penetration * 0.01)
-	var/soft_armor_modifier = min((1 - ((get_soft_armor(armor_type, def_zone) - penetration) * 0.01)), 1)
-	return clamp(((damage_amount - hard_armor_modifier) * soft_armor_modifier), 0, damage_amount)
+	penetration += sunder
```

Comment:
This massively buffs sunder. At the moment, sunder is a % reduction of armor. This change, as you said, makes it a flat AP increase, which in turn is a flat armor reduction. Due to this change, this buffs sunder against anything without 100 armor in a given field, which stays the same.

Example: Xeno has 25 armor against something. Lets suppose it has 25 sunder.
With the current code, this causes the 25 armor to be seen as 75% of its value, so 18.75.
With the new code, it removes a flag 25 armor, which means it gets considered **0** armor.

As this example should convey, this is a ridiculous increase to sunder effectiveness, scaling from "scuffed" to "why" the lower the base armor of the xeno is. (And, also just completely makes any existing display of sunder as % useless)